src/graph_compute/common.h

```diff
@@ -75,17 +92,6 @@
   #define cilk_sync
 #endif
 
-typedef uint64_t vid_t;  // vertex id type
-
-struct vertex_t {
-  vid_t id;
-  vid_t priority;
-  vid_t dependencies;
-  volatile vid_t satisfied;
-  vid_t cntEdges;
-  vid_t * edges;
-  double data;
-};
-typedef struct vertex_t vertex_t;
+//  typedef uint32_t vid_t;  // vertex id type
```

Comment:
oh, I had contemplated changing vid_t to a uint32_t, but it broke a bunch of code in the graph loading library which has constants in read routines which are specific to 8-byte ints.